Makefile

```diff
@@ -1,6 +1,51 @@
-# Makefile for Splunk Operator
+# VERSION defines the project version for the bundle.
+# Update this value when you upgrade the version of your project.
+# To re-generate a bundle for another specific version without changing the standard setup, you can:
+# - use the VERSION as arg of the bundle target (e.g make bundle VERSION=0.0.2)
+# - use environment variables to overwrite this value (e.g export VERSION=0.0.2)
+VERSION ?= 1.0.3
```

Comment:
this is part of SDK changes. they have removed version.go , now we are not hardcoding the version. when the build happens it will set version number. if the version number is not set then it will take VERSION from makefile. 
example 

make docker-build IMG=docker.io/splunk/splunk-operator:1.1.0 VERSION=1.1.0